samples/wifi/radio_test/sample_description.rst

```diff
@@ -175,6 +184,392 @@ Testing
               * R - Data rate (Mbps)
               * P - PHY overhead duration (us) (Values: 24 us - Legacy OFDM, 192 us - DSSS)
 
+
+         * To transmit a continuous tone, with the following configuration:
+
+           * Channel: 144
+           * TX power: 10 dBm
+           * Tone frequency: 2MHz
+
+           Execute the following sequence of commands:
+
+           .. code-block:: console
+
+              wifi_radio_test rx 0
+              wifi_radio_test tx 0
+              wifi_radio_test init 144
+              wifi_radio_test tx_power 10
+              wifi_radio_test tx_tone_freq 2
+              wifi_radio_test tx_tone 1
+
+
+
+         * To stop ongoing transmission of a continuous tone, use the following command:
+
+              wifi_radio_test tx_tone 0
+
+         .. note::
+
+            * wifi_radio_test tx_tone 0 should be only used if the command issued before this is wifi_radio_test tx_tone 1
+
+
+         * To enable DPD hardware block, use the following command:
+
+              wifi_radio_test dpd 1
+
+
+         * To disable DPD hardware block, use the following command:
+
+              wifi_radio_test dpd 0
+
+         .. note::
+
+            * DPD command can be used only during tone transmission. For TX packet transmission, control of DPD hardware
+            * is taken care by firmware.
```

Comment:
Rendered incorrectly as two bullets (one is expected).

Reply:
Done.